UIInfoSuite2/UIElements/ShowItemHoverInformation.cs

```diff
@@ -229,7 +231,12 @@ private void DrawAdvancedTooltip()
 
                 foreach (var requiredBundle in _prunedRequiredBundles)
                 {
-                    if (requiredBundle.Value.Contains(_hoverItem.Value.ParentSheetIndex)
+                    int quality = _hoverItem.Value is StardewValley.Object ? ((StardewValley.Object)_hoverItem.Value).Quality : 0;
+                    if (requiredBundle.Value.Any(
+                            itemQuality => 
+                                itemQuality.Key == _hoverItem.Value.ParentSheetIndex
+                                && (quality == 0 || quality >= itemQuality.Value)
```

Comment:
I don't know why this _quality == 0_ is responsible for here, but now your modifications are not working as intended.

Currently if I hover on an item which has **no stars**, it still writes I can add it to the Quality crops bundle because of this _quality == 0_. I don't want to modify it, because I don't know what was your original reason, so please ckeck it if it's needed :)

Reply:
I've uploaded a fix for this.